### PR TITLE
support select '' as a from order

### LINKED ISSUE #6541: support select '' as a from order

### PR DESCRIPTION
**Pull request by cmonkey**
Fixes #6541

Changes proposed in this pull request:
-
-
-

### CODE REVIEW

File: shardingsphere-sql-parser/shardingsphere-sql-parser-statement/pom.xml [hunk 1]
```diff
@@ -34,5 +34,9 @@
             <artifactId>shardingsphere-sql-parser-spi</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
```
Comment (tristaZero) [inline shardingsphere-sql-parser/shardingsphere-sql-parser-statement/pom.xml:38]: Is it necessary?

File: shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/util/SQLUtil.java [hunk 2]
```diff
@@ -95,6 +95,9 @@ public static String getExpressionWithoutOutsideParentheses(final String value)
     
     private static int getParenthesesOffset(final String value) {
         int result = 0;
+        if(Strings.isNullOrEmpty(value)){
+            return result;
+        }
```
Comment (tristaZero) [inline shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/util/SQLUtil.java:100]: A unit test is expected to add in `org.apache.shardingsphere.sql.parser.sql.util.SQLUtilTest`.

**Comment by tristaZero** [comment]
@cmonkey Hi thanks for rapid change.
I tried code `Strings.isNullOrEmpty(value)` in `SQLUtil` and found `guava dependency` is unnecessary, could you give it a check?

image

**Comment by cmonkey** [comment]
> @cmonkey Hi thanks for rapid change.
> I tried code `Strings.isNullOrEmpty(value)` in `SQLUtil` and found `guava dependency` is unnecessary, could you give it a check?
> 
> image

ok

**Pull request merged**